.papr.sh

```diff
@@ -55,6 +55,7 @@ export PATH=$HOME/gopath/bin:$PATH
 export GOSRC=/$GOPATH/src/github.com/projectatomic/libpod
 
 ${PACKAGER} install -y \
+	      cri-o \
```

Comment:
can we just install conmon ?

Reply:
I don't think this has worked its way out to RHEL and Centos yet.